sdk/core/Azure.Core/src/Pipeline/RetryOptions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.Pipeline.Policies;
+
+namespace Azure.Core.Pipeline
+{
+    public class RetryOptions
+    {
+        /// <summary>
+        /// Gets or sets the maximum number of retry attempts before giving up.
+        /// </summary>
+        public int MaxRetries { get; set; } = 3;
+
+        /// <summary>
+        /// Gets or sets the timespan used as delay between the retries or as a base for exponential backoff.
+        /// </summary>
+        public TimeSpan Delay { get; set; } = TimeSpan.FromSeconds(0.8);
+
+        /// <summary>
+        /// Gets or sets maximum timespan to pause between requests.
+        /// </summary>
+        public TimeSpan MaxDelay { get; set; } = TimeSpan.FromMinutes(1);
```

Comment:
We may want to consider making this `MaximumDelay`.   I thought that the guidelines asked to avoid abbreviating.

Reply:
`Max` is pretty ubiquitous and widely used in the base class library I prefer to keep it.

Reply:
Yeah, Max is grandfathered.